test/jest-config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "bail": false,
+  "coverageReporters": ["json-summary", "lcov", "html"],
+  "testEnvironment": "node",
+  "testRegex": "test/.*(\\.|/|_)(test|spec)\\.js$",
```

Comment:
this testRegex is very lax, I think we should pin it down to something specific so that we don't accept all sorts of files